test/unit/tasks/backend_api_test.rb

```diff
@@ -2,30 +2,6 @@
 
 module Tasks
   class BackendApiTest < ActiveSupport::TestCase
-    class DestroyOrphans < ActiveSupport::TestCase
```

Comment:
I would feel safer extracting the changes/removals in `lib/tasks/backend_api.rake` and `test/unit/tasks/backend_api_test.rb` to its own PR 🤔

Reply:
https://github.com/3scale/porta/pull/2247